### DISCUSSION TITLE
Large icons after using symbols map and nerd font v3

### DISCUSSION THREAD

**Discussion by shauryagoel** [kovidgoyal/kitty#7119] [Q&A]
Hi, I upgraded Iosevka nerd font v2 to v3 and symbols disappeared in kitty. To fix this I downloaded the symbols only nerd font and used symbol_map to specify the unicode characters that I wanted. Now, the icons are appearing larger compared to when I was using nerd font v2. Had anybody else faced the issue?

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/faq/#some-special-symbols-are-rendered-small-truncated-in-kitty

**Reply by shauryagoel** [reply]
I have tried this but unfortunately, it makes the icons too small compared to v2.
<img width="36" alt="image" src="https://github.com/kovidgoyal/kitty/assets/6321314/f732479e-a165-4389-b166-f01f7d11f7e8"> with using narrow_symbols + nerd font v3
<img width="45" alt="image" src="https://github.com/kovidgoyal/kitty/assets/6321314/6ddcdeab-8c80-4e54-8987-3b4259203d78"> with using symbol_map + nerd font v3
<img width="40" alt="image" src="https://github.com/kovidgoyal/kitty/assets/6321314/556a6bfa-8cb6-488e-b2ef-f0f305edc09b"> with using nerd font v2

The Kitty version is the same in all 3 cases

**Reply by kovidgoyal** [reply]
That's the only control kitty gives you, beyond that its up to the fonts. Presumably the standalone font has symbols larger than the iosevka version. I suggest you symbol_map the iosevka version to ensure they are rendered from there.

**Reply by shauryagoel** [reply]
Unfortunately, iosevka also has the same symbol size. Thanks for your help, will keep looking for a solution.

**Reply by bagaze** [reply]
Did you find a solution?

**Reply by shauryagoel** [reply]
Unfortunately not. Still facing the same issue.

**Reply by bagaze** [reply]
I actually found something for this. I used what's documented in the notes there: https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

```
# Nerd Fonts v3.2.0

symbol_map U+e000-U+e00a,U+ea60-U+ebeb,U+e0a0-U+e0c8,U+e0ca,U+e0cc-U+e0d7,U+e200-U+e2a9,U+e300-U+e3e3,U+e5fa-U+e6b1,U+e700-U+e7c5,U+ed00-U+efc1,U+f000-U+f2ff,U+f000-U+f2e0,U+f300-U+f372,U+f400-U+f533,U+f0001-U+f1af0 Symbols Nerd Font Mono
```

But instead of using the **Symbols Nerd Font Mono**, I downloaded and choose the **JetBrainsMono Nerd Font Mono** that can be downloaded there: https://www.nerdfonts.com/font-downloads

```
symbol_map U+e000-U+e00a,U+ea60-U+ebeb,U+e0a0-U+e0c8,U+e0ca,U+e0cc-U+e0d7,U+e200-U+e2a9,U+e300-U+e3e3,U+e5fa-U+e6b1,U+e700-U+e7c5,U+ed00-U+efc1,U+f000-U+f2ff,U+f000-U+f2e0,U+f300-U+f372,U+f400-U+f533,U+f0001-U+f1af0 JetBrainsMono Nerd Font Mono
```

With this, the size of the symbols is ok :)

<img width="36" alt="Screenshot 2025-01-08 at 16 37 07" src="https://github.com/user-attachments/assets/6dc6b9d3-140b-4aa9-93c8-e6520e13ba37" />
